main.go

```diff
@@ -90,6 +94,8 @@ func init() {
 	cli.StringVar(&backlink, "backlink", "", "backlink (optional)")
 	cli.StringVar(&httpHost, "http-host", "", "HTTP host")
 	cli.StringVar(&httpAddr, "http-addr", ":80", "HTTP listen address")
+	cli.StringVar(&tlsCertificate, "certificate", "", "enable TLS using a custom certificate/key pair")
+	cli.StringVar(&tlsKey, "key", "", "enable TLS using a custom certificate/key pair")
```

Comment:
```suggestion
	cli.StringVar(&tlsCertificate, "tls-certificate", "", "enable TLS using a custom certificate/key pair")
	cli.StringVar(&tlsKey, "tls-key", "", "enable TLS using a custom certificate/key pair")
```

I feel name-spacing these